test/integration/traces_test.go

```diff
@@ -58,6 +63,8 @@ func testHTTPTracesCommon(t *testing.T, doTraceID bool) {
 	require.NotEmpty(t, parent.TraceID)
 	if doTraceID {
 		require.Equal(t, traceID, parent.TraceID)
+		children := trace.ChildrenOf(parentID)
```

Comment:
Can we also do a direct comparison for the parent.ParentID? We also seem to set the flags in our code above, would it be possible to check that those make it through correctly?

Reply:
Ignore my comment about the flags, I don't think we should be setting it manually.